recipes/geocat-comp/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+python -m pip install --no-deps --ignore-installed -vv .
```

Comment:
I'm willing to make this change. 
Is there a reason that python is being called like a {{ variable }}?

Reply:
I don't know the full rationale, but that's the [standard pip install suggestion](https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip#use-pip) in our docs. I left the `--ignore-installed` since you had it, but if you're not sure you need it, I would recommend just using the line suggested in the docs as is.

Reply:
Re rationale, I've been told (if I remember correctly) that calling "python" may sometimes not work as expected (e.g. calling the system, non-conda python or something like that) and calling it like {{ variable }} ensures the correct python is being called.

Reply:
Thanks, I'll make this change.

Reply:
Looks like the change is something I need to move into meta.yaml. so I'll be placing that line there, and removing build.sh entirely.